src/textual/widgets/_data_table.py

```diff
@@ -473,8 +473,9 @@ def __init__(
         name: str | None = None,
         id: str | None = None,
         classes: str | None = None,
+        disabled: bool = False,
```

Comment:
Other widgets received a docstring line about this new attribute but I see none was added for the `DataTable`. Is it because `DataTable` is missing the docstring for its attributes?

Reply:
Yeah, I think `DataTable` might need a last round of docstring review so I wasn't going to make it part of this PR.